lib/SILGen/SILGenPoly.cpp

```diff
@@ -3360,13 +3360,23 @@ static CanSILFunctionType buildWithoutActuallyEscapingThunkType(
 // SWIFT_ENABLE_TENSORFLOW
 /// Given a value, extracts all elements to `result` from this value if it's a
 /// tuple. Otherwise, add this value directly to `result`.
-static void extractAllElements(SILValue val, SILBuilder &builder,
+static void extractAllElements(SILValue val, SILLocation loc,
```

Comment:
Note: this utility is duplicated in `lib/SILOptimizer/Differentiation.cpp`.
It could be made a general, shared `SILBuilder` utility (doesn't depend on SILGen-specific constructs).